### PR TITLE
Update

### PR DESCRIPTION
**Pull request by 0MLG**

### CODE REVIEW

**Comment by dgarijo** [comment]
@0MLG for Assignemnt 4, you have to create a new folder following the instructions in the slides.
As it is, this is not a valid PR

**Comment by 0MLG** [comment]
Sorry, I fixed it. Thank you for the warning

**Comment by ocorcho** [comment]
Assignment 2 is not correct, but do not correct any more. 
Now @dgarijo can continue with hassignment 4.

File: Assignment4/0MLG-190260/task06.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1sPwKJXn5ri2rBxn1LWQ4G3ZiyBlzP9iC
+
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib
```
Comment (dgarijo) [inline Assignment4/0MLG-190260/task06.py]: Won't work in python. This is notebook magic

File: Assignment4/0MLG-190260/task06.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+g.add((ns.JaneSmith, VCARD.Family,  Literal("Smith")))
+# Visualize the results
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.5: Add UPM as the university where John Smith works**"""
+
+# TO DO
+g.add((ns.UPM, RDF.type, ns.University))
+g.add((ns.UPM, VCARD.AGENT, ns.JohnSmith))
```
Comment (dgarijo) [inline Assignment4/0MLG-190260/task06.py]: This is wrong. VCARD.Agent is not a property. If you cannot find an appropriate property, create your own in the ns namespace

**Review by dgarijo — CHANGES_REQUESTED** [review]
Small change request in exercise 6. Exercise 7 is perfect

File: Assignment4/0MLG-190260/task06.py (hunk 2 above)
Comment (0MLG) [reply Assignment4/0MLG-190260/task06.py]: @dgarijo thanks for the feedback! I just fixed it

**Pull request merged**